src/CommandingOfficers/Modifiers/CODefenseModifier.java

```diff
@@ -22,7 +22,7 @@ protected final void modifyUnits(Commander commander, ArrayList<UnitModel> model
   {
     for( UnitModel um : models )
     {
-      if( um.weaponModels != null )
+      if( um.weapons != null )
```

Comment:
This modifier only applies to units with weapons? I feel like we copied this from AttackModifier and forgot to remove a condition.

Reply:
...whoops?
Clearly, refactoring this was a good way of catching existing bugs.